spec/tree-view-package-spec.coffee

```diff
@@ -3847,6 +3847,23 @@ describe "TreeView", ->
     files = Array.from(element.querySelectorAll('.entries .file'))
     files.find((file) -> file.fileName.textContent is text)
 
+describe "Service provider", ->
+  [provideTreeView, treeView] = []
+  beforeEach ->
+    waitForPackageActivation()
+
+    runs ->
+      provideTreeView = atom.packages.getActivePackage('tree-view').mainModule.provideTreeView()
```

Comment:
Change the name of this to `treeViewService`, and move it below `treeView` (also re-order the variable declarations above).